### DISCUSSION TITLE
How to migration the existing theme to the workflow ?

### DISCUSSION THREAD

**Discussion by trungh13** [krjo/shopify-webpack-dev-workflow#15] [Q&A] [answered]
Hi @krjo , 
At first, thanks for the repo, It look really great and time-saving with good developer experience. 

I'm new to Liquid/Shopify in general.  I'm trying to use Debut theme (https://themes.shopify.com/themes/debut/styles/default) to this workflow but then seem the `theme.js/css` is the entry point and they seem are not watching to `bundle.template.js/css` file that bundled by webpack.

If we don't have `theme.js/css` then seem like it's not working :-?

Can you give me some instructions/hint to make it work ?

Here is example from my git https://github.com/trungh13/shopify-webpack-dev-workflow/tree/feat/debut-template

Thanks you!

**Comment by trungh13** [comment, marked as answer]
Just found out that `theme.liquid` is quite same as `index.html` of another app. 
But it would be nice if we can have some guide in README about this.

**Reply by krjo** [reply]
Shopify has a very in-depth documentation. This repo is not the appropriate place to look for instructions on how to build Shopify themes.
